src/scss/sections/_instagram.scss

```diff
@@ -0,0 +1,33 @@
+.instagram {
+	&--feed {
+		overflow: hidden;
+		padding: calc-em(5);
+
+		@include media-breakpoint-up(lg) {
+      padding: calc-em(5) 0;
+    }
+
+		&__left {
+			@include media-breakpoint-up(lg) {
+				margin-left: calc-em(15);
+				margin-right: calc-em(-20);
+			}
+		}
+
+		picture {
+			margin: 0;
+			display: inline-block;
+			position: relative;
+			border: calc-em(5) solid transparent;
+			width: 100%;
+
+			img {
+				width: 100%;
+				max-width: 100%;
+				-o-object-fit: cover;
+				object-fit: cover;
```

Comment:
the logic of cover clashes with max-width 100% because cover fill all the space to fill its container block, this means that in case of an horizontal image, the browser might need to have the width wider in order to fill 100% the height. 
we don't need max-width